src/components/ButtonForm/styles.module.scss

```diff
@@ -11,13 +11,13 @@
   margin: 20px auto;
 }
 
-.button:nth-of-type(2)::before {
+.is-divider::before {
   content: '';
   background: $gray-main;
   height: 2px;
   left: 0;
   position: relative;
-  display: block;
+  display: flex;
```

Comment:
changing. `block` for `flex` is not the conclusion i wanted, this works for any display type, the problem here is that you are using `display: block;` on html that you want to occupy one entire block line. But. at 2020 its even better too use grid to achieve that layout behaviour, but using flex should be enough and fine. The problem for me here is that you dont understand completly box system and layout